File: quote-azure-backend/src/Models/UserLike.cs
namespace QuoteAzureBackend.Models
{
    public class UserLike
    {
        public string Username { get; set; } = string.Empty;
        public int QuoteId { get; set; }
        public int Order { get; set; }
        public DateTime LikedAt { get; set; }
    }
}

File: quote-ovhc-backend/internal/models/userlike.go
package models

import "time"

type UserLike struct {
	ID        int       `json:"id" db:"id"`
	UserID    int       `json:"user_id" db:"user_id"`
	QuoteID   int       `json:"quote_id" db:"quote_id"`
	Order     int       `json:"order" db:"order_index"`
	CreatedAt time.Time `json:"created_at" db:"created_at"`
}
